_activities/forcesfriction.md

```diff
@@ -5,12 +5,27 @@ lang: en
 # Content
 #
 title: "Forces and Friction"
-title_fr:
+title_fr:"Forces et Friction"
 age: 8-12
-objective: "Learn how forces affects the motion of objects."
-objective_fr:
-summary:
-summary_fr:
+objective: "Learn how forces affect the motion of objects."
+objective_fr:"Apprendre comment que les forces influencent le mouvement d'un objet."
```

Comment:
May need a space after the colon and before the quotation marks (several times in the file)